Dev-5/Dev-5/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+
+namespace Dev_5
+{
+    enum Commands
+    {
+        exit,
+        addNewCar,
+        countTypes,
+        countAll,
+        averagePrice,
+        averagePriceByType
+    }
+    /// <summary>
+    /// Entry point of program
+    /// Class operates with cars in car house
+    /// </summary>
+    class Program
+    {
+        static void Main()
+        {
+            try
+            {
+                CommandControl commandControl = CommandControl.GetInstance();
+                Menu menu = Menu.GetInstance();
+                Commands command = 0;
+                bool endFlag = false;
+                while (endFlag == false)
+                {
+                    menu.Show();
+                    Commands.TryParse(Console.ReadLine(), out command);
+                    switch (command)
+                    {
+                        case Commands.exit:
+                            endFlag = true;
+                            break;
+                        case Commands.addNewCar:
+                            commandControl.AddNewCar();
```

Comment:
Console.WriteLine()… What for? At any case it has to be out of swich-case block, DRY